docs/api.rst

```diff
@@ -5093,6 +5093,50 @@ Dealing with History Events
    :statuscode 409: No user is logged in or one of the identifiers to delete did not correspond to an event in the DB or one of the identifiers was for the last event in the corresponding transaction hash.
    :statuscode 500: Internal rotki error
 
+
+Querying online events
+============================================
+
+.. http:post:: /api/(version)/history/events/query
+
+   Doing a POST on this endpoint will query latest online events for the given event type and save them in the DB
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/history/events/pull HTTP/1.1
+      Host: localhost:5042
```

Comment:
Url differ in the example from the one in the description

```suggestion
      POST /api/1/history/events/query HTTP/1.1
      Host: localhost:5042
```

Reply:
ah fuck I missed this one. Started with pull but then saw query in yours and liked it more so I adjusted.